features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/MetaData.jsx

```diff
@@ -0,0 +1,253 @@
+                                            defaultMessage='-'
+                                        />
+                                    </Typography>
+                                </React.Fragment>
+                            }
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={12} md={6} lg={4}>
+                        {/* Provider */}
+                        <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
+                            <FormattedMessage
+                                id='Apis.Details.NewOverview.MetaData.provider'
+                                defaultMessage='Provider'
+                            />
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={12} md={6} lg={8}>
+                        <Typography component='p' variant='body1'>
+                            {api.provider && <React.Fragment>{api.provider}</React.Fragment>}
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={12} md={6} lg={4}>
+                        <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
+                            <FormattedMessage
+                                id='Apis.Details.NewOverview.MetaData.context:'
+                                defaultMessage='Context:'
+                            />
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={12} md={6} lg={8}>
+                        <Typography component='p' variant='body1'>
+                            {api.context && <React.Fragment>{api.context}</React.Fragment>}
+                        </Typography>
+                    </Grid>
+                    {/* Version */}
+                    {api.apiType === API.CONSTS.API && (
+                        <React.Fragment>
+                            <Grid item xs={12} md={6} lg={4}>
+                                <Typography component='p' variant='subtitle2' className={parentClasses.subtitle}>
+                                    <FormattedMessage
+                                        id='Apis.Details.NewOverview.MetaData.version'
+                                        defaultMessage='Version'
+                                    />
+                                </Typography>
+                            </Grid>
+                            <Grid item xs={12} md={6} lg={8}>
+                                <Typography component='p' variant='body1'>
+                                    {api.version && <React.Fragment>{api.version}</React.Fragment>}
+                                </Typography>
+                            </Grid>
+                        </React.Fragment>
+                    )}
+                    {/* Type */}
+                    {api.apiType === API.CONSTS.APIProduct ? null : (
```

Comment:
we can use `api.isAPIProduct()` method to do this check

Reply:
Let's do a code refactoring later, I just reorganized the existing code for layout changes.